src/it/scala/org/zalando/spearheads/innkeeper/routes/AcceptanceSpecsHelper.scala

```diff
@@ -20,7 +20,7 @@ import scala.language.implicitConversions
  */
 object AcceptanceSpecsHelper extends ScalaFutures with Matchers {
 
-  private val baseUri = "http://localhost:8080"
+  val baseUri = "http://localhost:8080"
```

Comment:
if you think so, you should consider splitting it into more classes, instead of exposing private fields ;)